.github/linters/sun_checks.xml

```diff
@@ -100,11 +100,9 @@ https://www.oracle.com/java/technologies/javase/codeconventions-contents.html
 		<!-- Checks for Javadoc comments.                     -->
 		<!-- See https://checkstyle.org/config_javadoc.html -->
 		<module name="InvalidJavadocPosition" />
-		<module name="JavadocMethod" />
```

Comment:
I don't think we should be disabling these modules. Can you link to what they are responisble for?

Reply:
Yes, there is a bug in the linter for the javadoc and generic T variable, it is reported as missing despite it is there,
see also the javadoc of the PackageStoreFactory.java class.